l10n_it_fatturapa_pec/models/mail_thread.py

```diff
@@ -17,7 +17,7 @@
 
 FATTURAPA_IN_REGEX = '^(IT[a-zA-Z0-9]{11,16}|'\
                      '(?!IT)[A-Z]{2}[a-zA-Z0-9]{2,28})'\
-                     '_[a-zA-Z0-9]{1,5}[.](xml|XML|zip|ZIP)$'
+                     '_[a-zA-Z0-9]{1,5}\\.(xml|XML|zip|ZIP)(\\.(p7m|P7M))?$'
```

Comment:
da un test su regex, `\\.` non mi viene interpretato come un punto, mi accetta `[.]`, funziona comunque in pratica?

Reply:
facendo un po' di prove li prende entrambi, non mi è chiaro perchè su regex online no, magari cambia in python3?

Reply:
Provato con python2 e python3, stesso comportamento:
```
>>> import re
>>> re.search('^(IT[a-zA-Z0-9]{11,16}|(?!IT)[A-Z]{2}[a-zA-Z0-9]{2,28})_[a-zA-Z0-9]{1,5}\\.(xml|XML|zip|ZIP)(\\.(p7m|P7M))?$', 'SM03473_XXXXX.xml')
<_sre.SRE_Match object at 0x7ff7fee7dc00>
>>> re.search('^(IT[a-zA-Z0-9]{11,16}|(?!IT)[A-Z]{2}[a-zA-Z0-9]{2,28})_[a-zA-Z0-9]{1,5}\\.(xml|XML|zip|ZIP)(\\.(p7m|P7M))?$', 'SM03473_XXXXX.xml.p7m')
<_sre.SRE_Match object at 0x7ff7fee7dca8>
>>> re.search('^(IT[a-zA-Z0-9]{11,16}|(?!IT)[A-Z]{2}[a-zA-Z0-9]{2,28})_[a-zA-Z0-9]{1,5}\\.(xml|XML|zip|ZIP)(\\.(p7m|P7M))?$', 'SM03473_XXXXX.xml.P7M')
<_sre.SRE_Match object at 0x7ff7fee7dc00>
>>> re.search('^(IT[a-zA-Z0-9]{11,16}|(?!IT)[A-Z]{2}[a-zA-Z0-9]{2,28})_[a-zA-Z0-9]{1,5}\\.(xml|XML|zip|ZIP)(\\.(p7m|P7M))?$', 'SM03473_XXXXX.xm')
>>> re.search('^(IT[a-zA-Z0-9]{11,16}|(?!IT)[A-Z]{2}[a-zA-Z0-9]{2,28})_[a-zA-Z0-9]{1,5}\\.(xml|XML|zip|ZIP)(\\.(p7m|P7M))?$', 'SM03473_XXXXX.xml.p7')
>>> re.search('^(IT[a-zA-Z0-9]{11,16}|(?!IT)[A-Z]{2}[a-zA-Z0-9]{2,28})_[a-zA-Z0-9]{1,5}\\.(xml|XML|zip|ZIP)(\\.(p7m|P7M))?$', 'SM03473_XXXXX')
>>>
```

Reply:
Forse per sicurezza è meglio non escluderle

Reply:
@d-stragapede @sergiocorato così: https://github.com/OCA/l10n-italy/compare/b8f516731d10c1879c76097b16dc2ae7891bb496..0994b671e26bb2e4f8026e0c8c38284a8789a6dc

Reply:
per me ok (prende anche un file `IT02205170968_00B32.p7m.p7m`, ma non è rilevante)

Reply:
Vedi anche https://www.odoo-italia.org/groups/sviluppo-17/sviluppo-1311

Reply:
@eLBati si penso che potrebbe succedere che ci sia un file p7m senza estensione xml.
Per i test io uso https://www.debuggex.com/ e la parte \\. è ok, l'ho inoltre testato su python e lo accetta.